dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/bootstrap/DubboShutdownHook.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.config.bootstrap;
```

Comment:
Why move the package

Reply:
Currently, ```DubboShutdownHook``` is instantiated and registered in ```DubboBootstrap```. That's why make  ```DubboBootstrap``` invisible to others except ```DubboBootstrap```. We can consider it as a part of Bootstrap.

Reply:
`invisible`?

Reply:
In the same module and public method

Reply:
@AlbumenJ change DubboShutdownHook to a package type.
![2024-08-15 11_40_57-dubbo3 2 - dubbo-config-api_src_main_java_org_apache_dubbo_config_bootstrap_Dubb](https://github.com/user-attachments/assets/378e4ede-c678-4961-a06c-7e1b6aa3e1ba)

Reply:
@AlbumenJ Any new comments?

Reply:
> In the same module and public method

@AlbumenJ If any other concern, will recall the move-package-ops.

Reply:
Still don't know why